api/db-cdr/generate-cdr/csv/achilles_analysis.csv

```diff
@@ -22,6 +22,6 @@
 3006,"Distribution of age by gender","concept_id","gender_concept_id","","","","box","distribution"
 3101,"Gender Stratification","concept_id","ppi_sex_at_birth_concept_id",NULL,NULL,NULL,"pie","counts"
```

Comment:
I think go ahead and make the other analyses names like these. I see in the mocks that is what they use.

Reply:
Changed the names of analysis.